src/main/java/net/imagej/notebook/DefaultNotebookService.java

```diff
@@ -246,6 +260,65 @@ public NotebookTable methods(final Class<?> type, final String prefix) {
 		}
 		return table;
 	}
+	
+	@Override
+	@SuppressWarnings({"unchecked", "rawtypes"})
+	public DefaultDatasetView view(final RandomAccessibleInterval<?> source) {
```

Comment:
Please use interfaces, not implementations, for return types. Here, it should return `DatasetView`, not `DefaultDatasetView`.